services/context-box/server/domain/usecases/enqueue_configs.go

```diff
@@ -53,15 +87,15 @@ func (u *Usecases) EnqueueConfigs() error {
 		return fmt.Errorf("error while enqueuing configs: %w", err)
 	}
 
-	if !u.schedulerQueue.CompareElementNameList(u.schedulerLogQueue) {
-		log.Info().Msgf("Scheduler queue content changed to: %v", u.schedulerQueue.GetElementNames())
+	if !u.schedulerQueue.CompareElementnameList(u.schedulerLogQueue) {
+		log.Info().Msgf("Scheduler queue content changed to: %v", u.schedulerQueue.GetElementnames())
 	}
-	u.schedulerLogQueue = u.schedulerQueue.GetElementNames()
+	u.schedulerLogQueue = u.schedulerQueue.GetElementnames()
 
-	if !u.builderQueue.CompareElementNameList(u.builderLogQueue) {
-		log.Info().Msgf("Builder queue content changed to: %v", u.builderQueue.GetElementNames())
+	if !u.builderQueue.CompareElementnameList(u.builderLogQueue) {
+		log.Info().Msgf("Builder queue content changed to: %v", u.builderQueue.GetElementnames())
 	}
-	u.builderLogQueue = u.builderQueue.GetElementNames()
+	u.builderLogQueue = u.builderQueue.GetElementnames()
```

Comment:
why is the `name` lowercase is ?

Follow camelCase naming, `GetElementnames` -> `GetElementNames`

Reply:
`Elementname` appeared more appealing to me. Can we keep it like this if it's not too necessary to switch to `ElementName`.